awx/ui_next/src/screens/Template/Survey/SurveyListItem.jsx

```diff
@@ -73,19 +86,51 @@ function SurveyListItem({
           </Stack>
         </DataListAction>
         <DataListCheck
+          isDisabled={!canAddAndEditSurvey}
           checked={isChecked}
           onChange={onSelect}
           aria-labelledby="survey check"
         />
         <DataListItemCells
           dataListCells={[
             <DataListCell key="name">
-              <Link to={`survey/edit/${question.variable}`}>
-                {question.question_name}
-              </Link>
+              <>
+                <Link to={`survey/edit/${question.variable}`}>
+                  {question.question_name}
+                </Link>
+                {question.required && (
+                  <Required
+                    aria-label={i18n._(t`Required`)}
+                    className="pf-c-form__label-required"
+                    aria-hidden="true"
+                  >
+                    *
+                  </Required>
+                )}
+              </>
+            </DataListCell>,
+
+            <DataListCell key="type">
+              <Label>{i18n._(t`Type`)}</Label>
+              {question.type}
+            </DataListCell>,
+            <DataListCell key="default">
+              <Label>{i18n._(t`Default`)}</Label>
```

Comment:
Adding a colon to this as well would help its appearance. `Default:`

Reply:
> * Format of multiple choice answers is not clear. The user has no way of telling if they are supposed to write commas or newlines as delimiters.

Will add a tooltip to this field to let the user know each answer must be on its own line.  

> * There is also no way to set a multiselect Default with more than one answer; this should be a TextArea if you need newlines.. If the user attempts to use commas as delimiters, they are met with a 400 error and no further information.

@dsesami if the user uses new lines they should for each answer choice and each default answer they should see a chip for each answer choice
![Screen Shot 2020-04-06 at 10 35 02 AM](https://user-images.githubusercontent.com/39280967/78569951-4f08f700-77f2-11ea-83b6-c83367d7f7ce.png)